pages/help/[page].tsx

```diff
@@ -0,0 +1,114 @@
+import addSiteLocation from '../../content/help/addSiteLocation.md';
+import { withSites } from '~/lib/sites';
+import ReactMarkdown from 'react-markdown';
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { LinkProps } from 'next/link';
+
+export const pages: any = {
+  'add-site-location': addSiteLocation,
+};
+
+type MenuLinkProps = {
+  linkProps: LinkProps;
+  label: string;
+  currentPath: string;
+};
+
+const MenuLink: React.FC<MenuLinkProps> = ({
+  linkProps,
+  label,
+  currentPath,
+}) => {
+  const textColor =
+    linkProps.href === currentPath ? 'text-amber' : 'text-white';
+  return (
+    <Link {...linkProps}>
+      <a>
+        <div
+          className={`flex transform items-center rounded-md px-4 py-2 text-gray-600 transition-colors hover:text-gray-700`}
+        >
+          <span className={`align-center mx-4 flex-1 font-medium ${textColor}`}>
+            {label}
+          </span>
+        </div>
```

Comment:
could we get this lil dot to show up next to the selected menulink? 
<img width="248" alt="Screen Shot 2023-04-26 at 5 11 21 PM" src="https://user-images.githubusercontent.com/62641231/234714363-d2cd0913-149d-431c-913d-0521b0b5ded4.png">